### ISSUE TITLE
Dictionary Without Acronyms and Improved Blacklisting Options

### ISSUE THREAD

**Issue by dean-futo** [futo-org/android-keyboard#986]
### Use Cases

A user requests that we create a dictionary that excludes capitalized acronyms to avoid nonsensical suggestions. They also ask that we allow users to view the entire dictionary list and select words to blacklist, rather than depending on manual input or prediction selection, which can be tedious and inefficient.

### References

Email ticket #6129

**Comment by apricity106** [comment]
Joined GitHub to +1 this request. Acronyms consistently interfere with simple but frequently used words such as "and," "but," "so."

**Comment by slycordinator** [comment]
The OpenBoard dictionaries that FUTO uses has a "flags=" field, which is set to "flags=abbreviation" for many such terms. 

For instance:
The file "en_US_wordlist.combined.gz" from openboard, contains a text file of word entries. One line looks like
"word=km,f=155,flags=abbreviation,originalFreq=155"
And there are many similar.

From what I can see searching the code, it doesn't seem to be considering that particular flag for suggestions or filtering.

**Comment by J-SoG-YK** [comment]
What I find odd, is that FUTO keyboard seems to prioritizes acronyms/initialisms when autocorrecting, while swiping. Even when swiping slowly. Eventually, the learning algorithm tends to figure things out. But initially, if an acronyms/initialisms exists for a store gesture, that acronyms/initialisms is prioritized over any other suggestion. 

So if what Slycordinator says is true, how does this proritization take place? I would think that acronyms/initialisms would have a lower priority, not a higher one.

**Comment by Zvonimir-FUTO** [comment]
Implemented in v0.1.27

**Comment by uri13** [comment]
It still has the same problem: the keyboard always suggests acronyms, random short & annoying words. It's ruining the whole beautiful FUTO experience. 

How to fix it? Should we install the non experimental dictionary?

**Comment by abb128** [comment]
@uri13 The default (builtin) English dictionary has had most acronyms removed. If you downloaded a different one then it may be different but I suggest the builtin one.

**Comment by uri13** [comment]
> [@uri13](https://github.com/uri13) The default (builtin) English dictionary has had most acronyms removed. If you downloaded a different one then it may be different but I suggest the builtin one.

Yeah, it's mostly in different languages. I speak daily spanish, english, french and catalonian. So I had to add all the dictionaries, and therefore all the acronyms. I don't know what to do...
